stdlib/list.ml

```diff
@@ -87,6 +87,10 @@ let rec flatten = function
 
 let concat = flatten
 
+let rec intersperse sep = function
+  | ([] | [ _ ]) as l -> l
+  | x :: (_ :: _ as xs) -> x :: sep :: intersperse sep xs
```

Comment:
Probably better to make the function tail-recursive.

Reply:
Done :slightly_smiling_face: